packages/ember-glimmer/tests/integration/components/curly-components-test.js

```diff
@@ -956,4 +957,49 @@ moduleFor('Components test: curly components', class extends RenderingTest {
 
     this.assertComponentElement(this.firstChild, { content: 'true' });
   }
+
+  ['@htmlbars a two way binding flows upstream through a CP']() {
+    let component;
+    let FooBarComponent = Component.extend({
+      init() {
+        this._super(...arguments);
+        component = this;
+      },
+
+      bar: computed({
+        get() {
+          return this._bar;
+        },
+
+        set(key, value) {
+          this._bar = value;
+          return this._bar;
+        }
+      })
+    });
+
+    this.registerComponent('foo-bar', {
+      ComponentClass: FooBarComponent,
+
+      template: '{{bar}}'
```

Comment:
That the CP is consumed by the template is different than in https://github.com/emberjs/ember.js/pull/11225